### ISSUE TITLE
Helm chart uses old manifest versions

### ISSUE THREAD

**Issue by izgeri** [cyberark/conjur-oss-helm-chart#34]
When attempted to deploy the helm chart, got an error about the deployment version. 

```
C:\Users\username\Downloads>C:\Users\username\Downloads\helm-v3.0.2-windows-amd64\windows-amd64\helm.exe install --set dataKey="$(docker run --rm cyberark/conjur data-key generate )"  C:\Users\sthiyagara\Downloads\conjur-oss-1.3.7.tgz --generate-name

Error: unable to build kubernetes objects from release manifest: unable to recognize "": no matches for kind "Deployment" in version "apps/v1beta2"
```

Need to investigate the correct API version to use for Deployment to support the needed K8s versions for our customers, and upgrade the manifests used by the helm chart.

**Comment by sgnn7** [comment]
Closing as fixed already (by https://github.com/cyberark/conjur-oss-helm-chart/pull/33)

**Comment by izgeri** [comment]
This is actually fixed by #33 - we still need a new release with these changes, and until the release this card will remain open

**Comment by izgeri** [comment]
The changes to fix this issue were included in https://github.com/cyberark/conjur-oss-helm-chart/releases/tag/v1.3.8
